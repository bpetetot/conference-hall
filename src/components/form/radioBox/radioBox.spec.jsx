/* eslint-env jest */
import React from 'react'
import RadioBox from './index'

const props = {
  value: 'value',
  name: 'name',
}

describe('components/radiobox', () => {
  it('should render', () => {
    expect(<RadioBox {...props}>children</RadioBox>).toMatchSnapshot()
  })
})
