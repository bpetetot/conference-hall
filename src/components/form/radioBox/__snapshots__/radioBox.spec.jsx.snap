// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/radiobox should render 1`] = `
<RadioBox
  checked={false}
  name="name"
  value="value"
>
  children
</RadioBox>
`;
